core/src/main/java/org/bitcoinj/core/DarkSendPool.java

```diff
@@ -1,230 +0,0 @@
-package org.bitcoinj.core;
-
-import org.bitcoinj.script.Script;
-import org.bitcoinj.utils.ContextPropagatingThreadFactory;
-import org.bitcoinj.utils.Threading;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import java.security.SecureRandom;
-import java.util.ArrayList;
-import java.util.concurrent.locks.ReentrantLock;
-
-/**
- * Created by Eric on 2/8/2015.
- */
-@Deprecated
-public class DarkSendPool {
```

Comment:
This is not used. Was intended as `CoinJoinStatusUpdate` perhaps? Seems safe to remove.

Reply:
This was added many years ago.  it was used as a background thread to sync the older format of masternode lists, sporks, governance objects (proposals and votes).  Slowly, changes were made to handle these operations in other places.  Masternode Lists synchronization was changed significantly.  Sporks are synced when connected to new peers.  

Last year, this background thread was moved to `Context` using `ScheduledExecutorService` 
https://github.com/dashevo/dashj/commit/ca16fdbbb020bbf82c36c45b6c1a202e1484309c#diff-46be2f806a625e71f6d79476b0f0cf41a615bc1ffdb6911582bc3afc03602549

We only start use `ScheduledExecutorService` if we want to sync governance objects, which we don't do with Dash Wallet.  This eliminates a background thread that wasn't doing anything.

Syncing governance objects would allow Dash Wallet or some other app (we were once working on something called Dash Control) to be set up as masternode operator.  In such a case, the app needs to obtain all the proposals and votes from other masternodes to present a view to the user of the proposals similar to what https://dashcentral.org/budget does.  Then the user could cast votes on proposals.  In our development the furthest we got was obtaining the list of votes and proposals (which took a long time).  At this point, the feature is probably broken.

Reply:
In short, we don't need this file anymore.